sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -142,10 +148,11 @@ case class Like(left: Expression, right: Expression) extends StringRegexExpressi
     } else {
       val pattern = ctx.freshName("pattern")
       val rightStr = ctx.freshName("rightStr")
+      val escapeChar = escapeCharOpt.getOrElse("""\\\\""")
```

Comment:
Have you checked if the other meta characters (e.g., ") work correctly?

Reply:
Yes. I try to use ", it not works fine.